apps/feature/index.js

```diff
@@ -4,6 +4,7 @@
 const GetFormSession = require('./behaviours/get-form-session');
 const ContinueForm = require('./behaviours/continue-form');
 const SaveFormSession = require('./behaviours/save-form-session');
+const areYouSure = require('./behaviours/are-you-sure');
```

Comment:
The others are capitalised, so keep it consistent with the others

Reply:
fixed this